docs/source/qstartguide/index.rst

```diff
@@ -45,32 +45,32 @@ Description of most used commands
       - Short description
     * - **expid**
       - Creates a new experiment and generates a new entry in the database by giving it a serial id composed of 4 letters. In addition, it also creates the folder experiment and the basic folder structure.
-    * - **create <expid>**
+    * - **create <EXPID>**
       - Generates the experiment workflow.
-    * - **run <expid>**
+    * - **run <EXPID>**
       - Runs the experiment workflow.
-    * - **monitor <expid>**
+    * - **monitor <EXPID>**
       - Shows the experiment workflow structure and status.
-    * - **inspect <expid>**
+    * - **inspect <EXPID>**
       - Generates Autosubmit scripts and batch scripts for inspection, by processing the tasks’ templates with the experiment parameters.
-    * - **refresh <expid>**
+    * - **refresh <EXPID>**
       - Updates the project directory.
-    * - **recovery <expid>**
+    * - **recovery <EXPID>**
       - Recovers the experiment workflow obtaining the last run complete jobs.
-    * - **setstatus <expid>**
+    * - **setstatus <EXPID>**
       - Sets one or multiple jobs status to a given value.
 
 
 Create a new experiment
 =======================
 
-    ``autosubmit expid -dm -H "local" -d "Tutorial"``
+.. runcmd:: autosubmit expid -dm -H "local" -d "Tutorial"``
```

Comment:
The rendered text shows the extra \`\` chars at the end... I think they can/have to be deleted?